coremltools/converters/mil/frontend/torch/ops.py

```diff
@@ -4357,3 +4408,162 @@ def scatter_add(context, node):
+    iou_threshold = inputs[2]
+    boxes = mb.expand_dims(x=boxes, axes=[0])
+    scores = mb.expand_dims(x=scores, axes=[0, -1])
+
+    # Follow tensorflow op example: TensorFlow's default value for score_threshold, Core ML does not
+    # have float('-inf') support, converted to minimum float32 instead
+    score_threshold = -3.4e38
+
+    _, _, x, _ = mb.non_maximum_suppression(
+        boxes=boxes,
+        scores=scores,
+        iou_threshold=iou_threshold,
+        score_threshold=score_threshold,
+        max_boxes=max_boxes
+    )
+
+    if not is_symbolic(num_boxes):
+        x = mb.squeeze(x=x, axes=[0])
+        x = mb.slice_by_index(x=x, begin=[0], end=[max_boxes], name=node.name)
+    else:
+        x = mb.squeeze(x=x, axes=[0], name=node.name)
+    context.add(x, torch_name=node.name)
+
+@register_torch_op
+def repeat_interleave(context, node):
+    inputs = _get_inputs(context, node)
+
+    x = inputs[0]
+    reps = inputs[1]
+    dim = inputs[2] if inputs[2] else 0
+
+    perm = [] + [axis for axis in range(x.rank) if axis not in []]
+
+    x = mb.transpose(x=x, perm=perm)  # torch.transpose(x, 0, 1)
+    x = mb.tile(x=x, reps=reps.val[0], name=node.name)  # torch.repeat(x, size)
+    x = mb.reshape(x=x, shape=(-1, x.shape[0]))  # x.view(-1, 2)
+    x = mb.transpose(x=x, perm=(-1, 0))  # torch.transpose(x, 0, 1)
+    dims = list(x.shape)
+
+    # Implementation of flatten
+    total = 1
+    start_val = dim
+    end_val = -1
+    start = len(dims) + start_val if start_val < 0 else start_val
+    end = len(dims) + end_val if end_val < 0 else end_val
+
+    if start > len(dims) or end > len(dims) or start < 0 or end < 0:
+        raise ValueError(
+            "Invalid start and end. (start, end) == ({}, {})".format(start, end_val)
+        )
+    if start > end:
+        raise ValueError(
+            "Start must be before end. (start, end) == ({}, {})".format(start, end_val)
+        )
+    x_shape = mb.shape(x=x)
+
+    shape1 = mb.slice_by_index(x=x_shape, begin=[0], end=[start])
+    shape2 = mb.slice_by_index(x=x_shape, begin=[end + 1], end=[len(dims)])
+
+    flatten_dim = -1
+    if not any_symbolic(x.shape):
+        flatten_dim = 1
+        for dim in dims[start: end + 1]:
+            flatten_dim *= dim
+
+    shape = mb.concat(values=(shape1, [flatten_dim], shape2), axis=0)
+    shape = mb.cast(x=shape, dtype="int32")
+    reshape = mb.reshape(x=x, shape=shape, name=node.name)
+
+    context.add(reshape, node.name)
+
+@register_torch_op(override=True)
+def narrow(context, node):
+    data, dim, start, length = _get_inputs(context, node, expected=4)
+    data_shape = mb.shape(x=data).val
+    begin = [0]*len(data_shape)
+    end = [x for x in data_shape]
+    begin[dim.val] = start.val
+    end[dim.val] = start.val+length.val
+    out = mb.slice_by_index(x=data, begin=begin, end=end)
+    context.add(out, torch_name=node.name)
+
+@register_torch_op(torch_alias=["__and_", '__and__'])
+def logicaland(context, node):
```

Comment:
`logical_and` was already implemented.
We need this function?

Reply:
@fukatani Awesome! Thanks, I've removed the op and used the logical_and version  and added the torch alias there.

A note on the alias `__and_`. Within the model the RPN requires the op
```
item = torch.__and__(torch.gt(widths0, 0.), torch.gt(heights0, 0.))
```
However `convert_nodes` assumes that ops ending in `_` are inplace ops and removes the trailing `_` before op lookup. Maybe I should just patch this issue instead and leave only the alias `__and__`.  yeah I think i'll do this

Reply:
Making this change as a separate PR sounds good.

Could someone provide a toy network with a `__and__` layer?